lib/src/main/java/com/diffplug/spotless/kotlin/DiKTatStep.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright 2021 DiffPlug
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.diffplug.spotless.kotlin;
+
+import com.diffplug.spotless.*;
+
+import java.io.IOException;
+import java.io.Serializable;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+import java.util.*;
+
+/** Wraps up [diktat](https://github.com/cqfn/diKTat) as a FormatterStep. */
+public class DiKTatStep {
```

Comment:
```suggestion
public class DiktatStep {
```
Fancy case is cool, but I think following coding standards is more important :)